src/Model_Training/Model_Training.py

```diff
@@ -8,7 +8,7 @@
 import yaml
 
 def run_model_training():
-    """Run model training using tensorflow
+    """  Run model training using tensorflow
```

Comment:
- [ ] It would be great to use either keras tuner or optuna to optimize through the hyper parameters
- [ ] As a starting point, I want to use F1 as a metric since this is an imbalanced dataset with a multi-class objective.